### ISSUE TITLE
Possible: URL escaping issue preventing searching of titles for metadata with non-alphanumeric characters

### ISSUE THREAD

**Issue by EazyCheeze1978** [sharkusmanch/playnite-pcgamingwiki-metadata-provider#28]
Searching on the full title of "The Signal from Tölva" returns no results, resulting in a (silently) failed search when downloading metadata. Several dozen other games in my library failed similarly during a bulk search. 

Searching on only "The Signal" on this one game resulted in a positive result, leading me to believe that perhaps it is due to the search parser failing to automatically "escape" the URLs correctly - insert the proper URL codes that represent special characters that cannot be typed normally in URLs. In the above example, the URL code that must be passed is %C3%B6 between capital T and lower case l, to render the o with diaeresis.

**Comment by sharkusmanch** [comment]
This one is proving tricky (always love some encoding trouble).

Am able to reproduce in my library by manually adding the game you mentioned, `The Signal from Tölva`.

Though, can't get the API to return the expected results in the search (even manually with curl). Will keep digging over the next few days.

https://github.com/sharkusmanch/playnite-pcgamingwiki-metadata-provider/blob/v1.0.5/src/PCGWClient.cs#L58=

**Comment by sharkusmanch** [comment]
Actually, looks like the REST library was correctly encoding the search. Problem was that these special characters are only recognized if the search string is quoted:

Unquoted:
```
$ curl -H "Content-Type: application/x-www-form-urlencoded; charset=utf-8" 'https://www.pcgamingwiki.com/w/api.php?action=query&list=search&srlimit=300&srwhat=title&srsearch=The%20Signal%20from%20T%C3%B6lva&format=json'
{"batchcomplete":"","query":{"searchinfo":{"totalhits":0},"search":[]}}
```

Quoted:
```
$ curl -H "Content-Type: application/x-www-form-urlencoded; charset=utf-8" 'https://www.pcgamingwiki.com/w/api.php?action=query&list=search&srlimit=300&srwhat=title&srsearch="The%20Signal%20from%20T%C3%B6lva"&format=json'
{"batchcomplete":"","query":{"searchinfo":{"totalhits":1},"search":[{"ns":0,"title":"The Signal From T\u00f6lva","pageid":58162,"size":10581,"wordcount":1136,"snippet":"|cover        = <span class='searchmatch'>The Signal From T\u00f6lva</span> cover.jpg\n...ersion now live, -50% sale on. - Steam Community :: Group Announcements :: <span class='searchmatch'>The Signal From T\u00f6lva</span>]&lt;/ref&gt;}}\n","timestamp":"2022-01-21T02:11:59Z"}]}}
```

**Comment by sharkusmanch** [comment]
Fixed in v1.0.5
